tests/e2e/server_cmd_test.go

```diff
@@ -15,7 +15,10 @@ func TestServerRunManualPanic(t *testing.T) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	go server.Run(ctx, conf)
```

Comment:
this was failing with:
```
fatal error: newproc: function arguments too large for new goroutine
```

Reply:
Weird, didn't know there was a goroutine limit. Would it be copying the config structure that's too large? But removing it changes the test, right?

Maybe try
```
go func() {
  server.Run(ctx, conf)
}()
```

Reply:
Yeah thats what I changed it to.  The error channel just carried over from debugging.